js-miniapp-bridge/src/types/screen.ts

```diff
@@ -0,0 +1,10 @@
+/** @internal */
+
+/**
+ * Screen orientation with lock action type.
+ */
+export enum ScreenAction {
```

Comment:
Is there the possibility we will have different types of "screen actions" in the future?

I was thinking it might be more clear to the user if this is named `ScreenOrientation`.